.env

```diff
@@ -24,7 +24,7 @@ REMME_CONSENSUS=
 # Minimum number of peers to start network synchonization
 REMME_MINIMUM_PEERS_CONNECTIVITY=1
 # List of methods which should be enabled on REMME REST API
-REMME_REST_API_AVAILABLE_METHODS=*
+REMME_REST_API_AVAILABLE_METHODS=/atomic-swap:GET;/batch_status:GET;/certificate/p12:POST;/certificate:POST;/node_key:GET;/pub_key:POST;/token:GET;/transaction:POST;/user:GET
```

Comment:
Why POST for pub_key? @apadenko

Reply:
POST is read operation for pub_key, same as for certificate